packages/openapi-v3/src/controller-spec.ts

```diff
@@ -93,12 +94,57 @@ function resolveControllerSpec(constructor: Function): ControllerSpec {
     if (!operationSpec) {
       // The operation was defined via @operation(verb, path) with no spec
       operationSpec = {
-        responses: {},
+        responses: {
+          '200': {
+            description: '',
```

Comment:
Let's use a non-emptyh default value please. For example, we can use the controller name and method name to build a more descriptive text:

```ts
description: `Return value of controller method ${fullMethodName}`
```